scripts/install/RegisterLegacyPortableLibraries.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2014 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoQtiItem\scripts\install;
+
+use oat\oatbox\extension\InstallAction;
+use oat\tao\model\ClientLibRegistry;
+
+/**
+ * Register former portable shared libraries to make existing PCI implementation compatible
+ */
+class RegisterLegacyPortableLibraries  extends InstallAction
+{
+    public function __invoke($params)
+    {
+        //register location of portable libs to legacy share lib aliases for backward compatibility
+        $portableSafeLibPath = ROOT_URL.'taoQtiItem/views/js/legacyPortableSharedLib';
```

Comment:
Can we still use `ROOT_URL` ?

Reply:
Hmm, I am checking its usage now.

Reply:
So I replaced the call of $clientLibRegistry->set() to $clientLibRegistry->register() to ensure only relative path is written in the client lib registry config file. Therefore the ROOT_URL usage is safe here as the ROOT_URL will now be replaced with the relative URL.
Example: 
this `ROOT_URL.'taoQtiItem/views/js/legacyPortableSharedLib/jquery_2_1_1'` becomes 
```
'IMSGlobal/jquery_2_1_1' => array(
            'extId' => 'taoQtiItem',
            'path' => 'js/legacyPortableSharedLib/jquery_2_1_1'
        ),
```

Reply:
No, my question was more about the usage of the constant itself. I think there was a task at some point to replace this constant. (please check with a backend)

Reply:
No it is ok. I have checked with the backend. No plan for removing it so far.